articles.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+---
+
+## Articles, Blog Posts & Presentations
+In the following page you will find different articles, blog posts and presentations ordered chronologically.
```

Comment:
This first sentence is a bit redundant.

I had linked to the corresponding [page](https://flatpak.org/presentations/) on Flatpak's website in the [issue](https://github.com/containers/containertoolbx.org/issues/24).  If you really want an introductory sentence, then use something like that showing the reader how to get new entries added to the list.

Either way, we still need a newline between the main title of the page and the first sentence.

Reply:
I changed the sentence to match the one on [flatpak.org](https://flatpak.org/presentations/).